plugins/rabbitmq.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: rabbitmq
+spec:
+  homepage: https://github.com/rabbitmq/cluster-operator
+  shortDescription: Manage RabbitMQ clusters
+  version: v0.48.0
+  description: |
+    A plugin providing utilities to operate RabbitMQ clusters deployed
+    by the RabbitMQ cluster operator.
```

Comment:
I see that you offer to install the cluster operator via this tool. Maybe mention this as a post-install setup step here or in the `caveats` section. Usually we don't recommend to print basic usage instructions, but this is different, because the plugin only becomes useful once the operator is installed.

Reply:
I added a `caveats` section.